crates/sui-core/src/transaction_driver/message_types.rs

```diff
@@ -19,19 +19,41 @@ use sui_types::{
     transaction::Transaction,
 };
 
-#[derive(Serialize, Deserialize, Clone, Debug)]
+#[derive(Clone, Debug)]
 pub struct SubmitTxRequest {
-    pub transaction: Transaction,
+    pub transaction: Option<Transaction>,
+    pub ping: Option<PingType>,
```

Comment:
I wonder if it is easier to keep `SubmitTxRequest` as it is, but introduce another function on `AuthorityAPI` for ping request?